cam0:
  cam_overlaps: [1, 2]
  camera_model: pinhole
  distortion_coeffs: [-0.0678890809361792, 0.007709008000713503, -0.010307689056822753,
    0.003975705353253985]
  distortion_model: equidistant
  intrinsics: [393.37990041643656, 395.3197707435412, 370.05019923180885, 239.38250128073372]
  resolution: [752, 480]
  rostopic: /gvio/cam0/image
cam1:
  T_cn_cnm1:
  - [0.9997999638114503, 0.00010281287341368757, -0.02000054479569596, -0.08720159399936726]
  - [-0.0002122414377615125, 0.9999850211073896, -0.005469233439385734, 0.0007968664529640217]
  - [0.019999682902078156, 0.0054723843391577135, 0.9997850097363181, -0.0007160979652175064]
  - [0.0, 0.0, 0.0, 1.0]
  cam_overlaps: [0, 2]
  camera_model: pinhole
  distortion_coeffs: [-0.0658404259251678, 0.003081171049227224, -0.004493863430556307,
    0.001097707477325051]
  distortion_model: equidistant
  intrinsics: [390.04316114278265, 391.78659099713224, 372.81266393927774, 227.2992327622048]
  resolution: [752, 480]
  rostopic: /gvio/cam1/image
cam2:
  T_cn_cnm1:
  - [0.9995732598809758, 0.023954113088115778, -0.016718211539807844, 0.05113511186382073]
  - [-0.024300322806877175, 0.999487836131887, -0.020822097298027958, -0.07432745478977369]
  - [0.016210874202508902, 0.02121946961091956, 0.9996434072543221, 0.011323941258338556]
  - [0.0, 0.0, 0.0, 1.0]
  cam_overlaps: [0, 1]
  camera_model: pinhole
  distortion_coeffs: [-0.11175399755160273, 0.006660995340753397, 0.02668405418703251,
    -0.034837898527287206]
  distortion_model: equidistant
  intrinsics: [524.9162413300063, 526.7772489140208, 364.9123198868755, 238.15760650709848]
  resolution: [752, 480]
  rostopic: /gvio/cam2/image
